src/r5py/r5/transport_network.py

```diff
@@ -110,7 +92,7 @@ def from_directory(cls, path):
             if util.contains_gtfs_data(potential_gtfs_file)
         ]
         try:
-            with open(os.path.join(str(path), "build.json")) as build_json_file:
+            with open(pathlib.Path(path) / "build.json") as build_json_file:
```

Comment:
Does using `/` for sure work on Windows? (historically creating paths just by adding `/` has been quite problematic).

Not sure how pathlib works because I haven't used it that much, so does it take care of that the filepaths work appropriately both in Unix + Windows? I have used `os.path.join()` function to tackle these things, but that might be "old style" and I'm just not aware of how it should be done nowadays.  

This comment applies to all places where this code style is used.

Reply:
that’s why pathlib - it standardises those things, and `/` is in fact an operator here, and not a character in the path

Reply:
Right! Good to update my old habits here also 🙂 Need to start using pathlib more.